misc/identity-keys/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@walletconnect/identity-keys",
-  "version": "0.1.10",
+  "version": "0.12.0-8d867",
```

Comment:
Looks like we jumped a few versions here

Reply:
that is weird, I'll fix that up